collect_app/src/main/java/org/odk/collect/android/widgets/WidgetFactory.java

```diff
@@ -122,6 +124,8 @@ public static QuestionWidget createWidgetFromPrompt(FormEntryPrompt prompt, Cont
                             questionWidget = new StringNumberWidget(context, questionDetails, readOnlyOverride);
                         } else if (appearance.equals(WidgetAppearanceUtils.URL)) {
                             questionWidget = new UrlWidget(context, questionDetails, new CustomTabHelper());
+
+                            analytics.logEvent(URL_WIDGET, "UrlWidget", questionDetails.getFormAnalyticsID());
```

Comment:
I think the action here should just be `"Url"`. What we care about is that the user saw a url question.